src/bin/vip-whoami.ts

```diff
@@ -13,31 +12,33 @@
  * Internal dependencies
  */
 import { getCurrentUserInfo } from '../lib/api/user';
+import { User } from '../graphqlTypes';
 import command from '../lib/cli/command';
 import { trackEvent } from '../lib/tracker';
 import * as exit from '../lib/cli/exit';
 
 export async function whoamiCommand() {
-	const trackingParams = {
+	const trackingParams: { command: string } = {
 		command: 'vip whoami',
 	};
 
 	await trackEvent( 'whoami_command_execute', trackingParams );
 
-	let currentUser = {};
+	let currentUser: User;
 	try {
 		currentUser = await getCurrentUserInfo();
-	} catch ( err ) {
-		await trackEvent( 'whoami_command_error', { ...trackingParams, error: err.message } );
+	} catch ( err: unknown ) { // eslint-disable-line
```

Comment:
```suggestion
	} catch ( err: unknown ) {
```

I think it is safe to remove now :-)